x-pack/plugin/eql/src/main/antlr/EqlBase.g4

```diff
@@ -199,6 +199,7 @@ STRING
     | '"'   ('\\' [btnfr"'\\] | ~[\r\n"\\])* '"'
     | '?"'  ('\\"' |~["\r\n])* '"'
     | '?\'' ('\\\'' |~['\r\n])* '\''
+    | '"""' ('\\"' |~['\r\n])* '"""'
```

Comment:
This sttill doesn't seem right and needs more tests.
Under this regex, you start and end with `"""`, but you can't have any `'` characters in between. In addition, you can have an infinite number of `"` characters. This would mean `"""""""""""""""` is valid syntax. But I'm not sure that's good behavior.